README.md

```diff
@@ -51,25 +39,35 @@ my_params_dict = my_params.dict(by_alias=True)
 print('Initial temperature of cell:', my_params_dict["Parameterisation"]["Cell"]["Initial temperature [K]"])
 ```
 
-If you want to pretty print the entire object, you can use the `devtools` package to do this (remember to `pip install devtools`)
-
+The entire BPX object can be pretty printed using the `devtools` package 
 ```python
 from devtools import pprint
 pprint(my_params)
 ```
 
 You can convert any `Function` objects in `BPX` to regular callable Python functions, for example:
-
 ```python
 positive_electrode_diffusivity = my_params.parameterisation.positive_electrode.diffusivity.to_python_function()
 diff_at_one = positive_electrode_diffusivity(1.0)
 print('positive electrode diffusivity at x = 1.0:', diff_at_one)
 ```
 
 If you want to output the complete JSON schema in order to build a custom tool yourself, you can do so:
-
 ```python
 print(bpx.BPX.schema_json(indent=2))
 ```
 
 According to the `pydantic` docs, the generated schemas are compliant with the specifications: JSON Schema Core, JSON Schema Validation and OpenAPI.
+
+## 📖 Documentation
+API documentation for the `bpx` package can be built locally using [Sphinx](https://www.sphinx-doc.org/en/master/). To build the documentation first [clone the repository](https://github.com/git-guides/git-clone), then run the following command:
```

Comment:
This didn't work straight away for me ('No module named bpx' error). I just git-cloned the repo (dev branch) and tried the command below.

Reply:
you have to install bpx first, will update the instructions

Reply:
yeah, I realised that I had installed the previous version of BPX, that's why it probably complained.